src/osm_configurator/view/toplevelframes/calculation_frame.py

```diff
@@ -432,13 +434,14 @@ def __get_next_phase(self, current_phase: CalculationPhase) -> CalculationPhase:
                 take_next = True
         return CalculationPhase.NONE
 
+    @staticmethod
```

Comment:
ok here it is even weirder, you say it is a static method, but you keep the self?

Reply:
ok